File: PYTHON/PRAK402-2210817220029-SitiAinurRahmawati.py
angka=int(input())
for i in range (0, angka+1, 1) :
    if i%2!=0 :
        print(i, end=" ")
    else :
        continue
print()
for i in range (angka, 0, -1) :
    if i%2==0 :
        print(i, end=" ")
        continue
    else :
        continue

File: C/PRAK402-2210817220029-SitiAinurRahmawati.c
#include <stdio.h>
int main () {
    int angka, i;
    scanf("%d", &angka);
    for(i=1;i<=angka;i++){
        if (i%2!=0) {
            printf("%d ", i);}
        else {
            continue;} 
    } 
    printf("\n");
    for(i=angka;i>0;i--){
        if (i%2==0) {
            printf("%d ", i);}
        else {
            continue;}
    }
}